src/views/common.js

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components'
+
+export const Page = styled.div`
+  width: 100%;
+`
+
+export const Content = styled.div`
+	width: auto;
+			
+	display: flex;
+	flex-direction: column;
+	gap: 2rem;
+
+	margin: 2rem 5rem;
```

Comment:
Acho que a margem dos lados ta muito grande, mudar para 

```suggestion
	margin: 3.5rem 2.5rem;
```